core/src/main/java/org/apache/iceberg/SerializableTable.java

```diff
@@ -120,7 +120,7 @@ private FileIO fileIO(Table table) {
     return table.io();
   }
 
-  private Table lazyTable() {
+  protected Table lazyTable() {
```

Comment:
`protected` is for test purpose. Should we add `@VisibleForTesting` or similar (just as indication) ?

Reply:
this method isn't used for test purposes, but rather to get access to the underlying table in the subclass

Reply:
Oh yes, right. I misread. Good to me then.